src/kajal/Codingexam/ReturnIndextarget.java

```diff
@@ -0,0 +1,24 @@
+package kajal.Codingexam;
+
+public class ReturnIndextarget {
+	int num1=0;
+	int num2=0;
+
+	void checkIndex(int[] nums,int targetNum) {
+		for(int index=0;index<nums.length;index++) {
+			for(int innerIndex=0;innerIndex<nums.length;innerIndex++)
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files